processor/src/test/java/org/mapstruct/ap/test/inheritfromconfig/InheritFromConfigTest.java

```diff
@@ -141,6 +141,35 @@ public void explicitInheritedMappingIsAppliedInReverseDirectlyFromConfig() {
     }
 
     @Test
+    @IssueKey( "1255" )
+    @WithClasses({ CarMapperReverseWithAutoInheritance.class, AutoInheritedReverseConfig.class } )
+    public void autoInheritedMappingIsAppliedInReverseDirectlyFromConfig() {
```

Comment:
I think that we need to add the reverse method to the `CarMapperReverseWithAutoInheritance` and validate that the inheritance is not inherited